content/en/docs/reference/kubernetes-api/ref/cluster/binding-v1.md

```diff
@@ -0,0 +1,149 @@
+---
+api_metadata:
+  apiVersion: "v1"
+  import: "k8s.io/api/core/v1"
+  kind: "Binding"
+collapsible: false
+content_type: "api_reference"
+description: "Binding ties one object to another; for example, a pod is bound to a node by a scheduler."
+draft: false
+title: "Binding"
+weight: 9
+---
+
+`apiVersion: v1`
+
+`import "k8s.io/api/core/v1"`
+
+
+## Binding
```

Comment:
@feloy The title is `Binding` and the first H2 section is the same name.
Do you need this heading?

Reply:
Generally, a page contains several resources related to a "main" resource.

For example, the Node page (http://localhost:1313/docs/reference/kubernetes-api/cluster-resources/node-v1/) contains Node, NodeSpec, NodeStatus, NodeList, all related to the Node main resource.

The top `Node` is related to the main resource, tht second `Node` is related to the specific Node resource (at the same level as NodeSpec, ...). 

For Binding, there are no related BindingSpec, ... resources, so we could change the layout to describe directly the Binding resource.

Do we prefer to be generic between all pages, or try to adapt depending on these specific cases?